Apps/Src/Task_FaultState.c

```diff
@@ -0,0 +1,119 @@
+/* Copyright (c) 2020 UT Longhorn Racing Solar */
+#include "os.h"
+#include "Tasks.h"
+#include "BSP_Contactor.h"
+#include "BSP_Lights.h"
+#include "Voltage.h"
+#include "BSP_Fans.h"
+#include "Temperature.h"
+#include "Amps.h"
+#include "EEPROM.h"
+#include "BSP_WDTimer.h"
+#include "CLI.h"
+#include "CANbus.h"
+#include "BSP_UART.h"
+#include "config.h"
+
+/*
+ * Note: do not call this directly if it can be helped.
+ * Instead, call an RTOS function to unblock the mutex
+ * that the Fault Task is pending on.
+ */
+void EnterFaultState() {
+    // Turn Contactor Off
+    BSP_Contactor_Init();
+    BSP_Contactor_Off();
+    //Set Fans to full speed
+    BSP_Fans_Init();
+    BSP_Fans_Set(1, 8);
+    BSP_Fans_Set(2, 8);
+    BSP_Fans_Set(3, 8);
+    BSP_Fans_Set(4, 8);
+    // Turn Strobe Light On
+    // Turn LEDs On and logs Error into EEPROM
+    BSP_Lights_Init();
+    BSP_Light_Off(RUN); //turn off run light
+    BSP_Light_On(FAULT);
+    EEPROM_Init();
+    if (BSP_WDTimer_DidSystemReset()) Fault_BitMap = Fault_WDOG;
```

Comment:
Make this a bit more readable.
```suggestion
    BSP_Lights_Init();
    BSP_Light_Off(RUN); //turn off run light
    BSP_Light_On(FAULT);
    
    EEPROM_Init();
    
    // Check watchdog status
    if (BSP_WDTimer_DidSystemReset()) {
        Fault_BitMap = Fault_WDOG;
    }
```

Reply:
Updated in #428